tests/h/search/index_test.py

```diff
@@ -1,18 +1,21 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from datetime import datetime
+
 import mock
 import pytest
 
-import elasticsearch
```

Comment:
It looks like this is a requirement that wasn’t needed, and this is just a linting change. Is that the case? If so, I think it should be in a separate commit to avoid confusion.